manifests/init.pp

```diff
@@ -162,11 +162,22 @@
 
   include certs::pulp_client
```

Comment:
Unrelated, but do we still need this?

Reply:
In theory yes, for N-1 testing in devel environments this could be useful.